manual/create-new-environment.md

```diff
@@ -0,0 +1,58 @@
+# Create a new environment
+This is a guide to creating a new tdr environment in a clean AWS account.
+
+## Create parameters in parameter store.
+There are five parameters which need to be set as they can't be created or found by terraform
+* /mgmt/aws_elb_account - The parent account where the logs are sent. Maintained by digital services so no way to have this in terraform
+* /mgmt/cost_centre - This is a TNA property which is not available to terraform
+* /mgmt/management_account - There's no way to get this while running terraform for other accounts
+* /mgmt/trusted_ips - The IP list for the WAF. We could probably reconfigure this to come from `tdr-configurations` but for now, this needs to be set.
+* /prod/frontend/auth/thumbprint - The thumbprint used for the IAM external provider. 
+
+### External provider thumbprint
+The OIDC provider thumbprint comes from an AWS provided certificate and is the same for anything using their certificates in a region.
+You can find it by running:
+
+`openssl s_client -servername oidc.eks.eu-west-2.amazonaws.com -showcerts -connect oidc.eks.eu-west-2.amazonaws.com:443`
+
+Copy the final certificate that is output starting from `-----BEGIN CERTIFICATE-----` up to `-----END CERTIFICATE-----` inclusive and write to a file certificate.crt.
+
+`openssl x509 -in certificate.crt -fingerprint -noout`
+
+Remove the colons from the output of this command and put it in lower case, this is your thumbprint.
+
+## Run terraform
+`terraform apply -var 'tdr_account_number=account_number`
```

Comment:
Do you need to switch to a different workspace and run `terraform init` first? Probably worth including for the avoidance of doubt. Or link to the instructions in tdr-terraform-environments.

Reply:
I've added in terraform workspace select and terraform init.